TFEngine.py

```diff
@@ -1427,8 +1464,24 @@ def extra_fetches_callback(seq_tags, **extra_fetches_out):
     if output_per_seq_file:
       print('Write eval results per seq to %r' % output_per_seq_file, file=log.v3)
       from Util import betterRepr
+
+      # write fetches to file
       with open(output_per_seq_file, 'w') as f:
-        f.write(betterRepr(results_per_seq) + '\n')
+        def create_output_string(st):
+          """
+          Creates a colon (;) separated output string in the order given by `requested_outputs`
+          :param str st : a seq_tag
```

Comment:
There is an extra space between `str st` and `:`. Why? Make it consistent. (I'm even not sure if this is allowed.)

Reply:
Also, instead of this simple description ("a seq_tag"), why don't you just call the variable like that? Then the comment would not be needed, and the remaining code becomes also easier to read.

Reply:
(But anyway, the whole function should be removed, as said below.)